src/main/java/com/keniu/repositories/BookRepository.java

```diff
@@ -2,10 +2,12 @@
 
 import com.keniu.models.Book;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.repository.PagingAndSortingRepository;
 
 /**
  * Repository interface for managing {@link Book} entities.
  */
-public interface BookRepository extends JpaRepository<Book, Long> {
+public interface BookRepository extends JpaRepository<Book, Long>,
+        PagingAndSortingRepository<Book, Long> {
```

Comment:
```suggestion
 {
```
because of the inheritance relationship, the JpaRepository contains the full API of CrudRepository and PagingAndSortingRepository.

Reply:
Fixed